demo/README.md

```diff
@@ -32,254 +30,67 @@ And now, let's run emp apps to see what apps we have running in our Empire envir
 ```
 
-----
+So that created an ECS service and attached an ELB to our new application.
 
-Empire makes it easy to deploy a Docker image as an application. To do this, we'll need to first push our image to a Docker registry, then deploy it using the Empire CLI.
+_Find ECS service and ELB_
 
-Let's start by pushing our image to the Docker registry:
+So, it looks like our application isn't running yet, we can see what processes are running using the `ps` command:
 
 ```console
-$ docker push ejholmes/app:latest
+$ emp ps -a www
 ```
 
-The `emp deploy` command is our primary interface to deploying docker images from a docker registry.
-
-```console
-$ emp help deploy
-```
+Great! We're up and running. But it looks like I made a typo and called the company Acem Inc. Let me go ahead and fix that real quick.
 
-Let's go ahead and create an app and deploy the latest tag of our Docker image.
+_Fix typo, build and deploy_
 
 ```console
-$ emp create app
-```
-
-As I mentioned before, apps are internal to the VPC by default, but we can expose it publicly by adding a domain to the app.
-
-```
-$ emp domain-add app -a app
-```
-
-And now we'll deploy the Docker image:
-
-```
-$ emp deploy ejholmes/app:latest
+$ docker build -t acmeinc/www .
+$ docker push acmeinc/www
+$ emp deploy 
 ```
 
-Empire pulled the Docker image, extracted the process types from the Procfile, and created the ECS services.
+When we deploy a new version, we'll see that Empire created a new release for us:
 
 ```console
-$ aws ecs list-services --cluster $CLUSTER
-$ export SERVICE=
+$ emp releases -a www
 ```
 
-We can see that we now have two additional ECS services for our app; one for the web process, and another for the worker process.
-
-We can also see that Empire created an ELB attached to the web process.
-
-```console
-$ aws ecs describe-services --service $SERVICE --cluster $CLUSTER --query 'services[0].loadBalancers[0]'
-$ export ELB_NAME=
-$ aws elb describe-load-balancers --load-balancer-names $ELB_NAME --query 'LoadBalancerDescriptions[0].DNSName'
-$ export ELB=
-```
-
-It'll take a minute or two for the ELB to begin resolving, so let's take a look at some of the other emp CLI commands that we can use.
-
-We can us the `ps` subcommand to list the running and pending processes for the application.
-
-```console
-$ emp ps -a app
-```
-
-We can see that we're running 1 instance of the web process.
-
-This just lists all of the ECS tasks for the ECS services that compose this application.
-
-```console
-$ aws ecs list-tasks --service $SERVICE --cluster $CLUSTER
-```
-
-Let's go ahead and scale up the worker process.
+When we deploy a new version of a web process, ECS will spin up new version of the app, wait for connections to drain from the existing version and then remove the old processes.
 
 ```console
-$ emp scale worker=1 -a app
+$ emp ps -a www
 ```
 
-In a minute or two, we should see a new process for the worker process.
+Now that looks better.
```

Comment:
I'm thinking it might be fun to do something like "Wait, I just got a text and apparently the CEO says they're going to provide Anvils as a Service and they need the app built now!" and then build a little Go app that provides an API for dropping anvils on Wile E. Coyote. Maybe talk about Procfile's and a worker process here?